GooglePlayInstant/Editor/AppBundlePublisher.cs

```diff
@@ -12,34 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if UNITY_2018_3_OR_NEWER
 using UnityEditor;
 using UnityEngine;
-#else
-using System;
-#endif
 
 namespace GooglePlayInstant.Editor
 {
     /// <summary>
-    /// Helper to build an Android App Bundle file suitable for publishing on Play Console.
+    /// Helper to build <a href="https://developer.android.com/platform/technology/app-bundle/">Android App Bundle</a>
+    /// files suitable for publishing on Play Console.
     /// </summary>
     public static class AppBundlePublisher
     {
         /// <summary>
-        /// Builds an Android App Bundle.
+        /// Builds an Android App Bundle at a user-specified file location.
         /// </summary>
         public static void Build()
         {
-#if UNITY_2018_3_OR_NEWER
+#if !UNITY_2018_3_OR_NEWER
+            if (!AndroidAssetPackagingTool.CheckConvert())
```

Comment:
Should we check for these tools after the user selects a path? That way we could continue building the app bundle after they download the tools, instead of just breaking out of the current action, and requiring them to hit "Build Android App Bundle..." again.

Reply:
That sounds ideal, but difficult to accomplish. These actions may need to display a UI, may be cancelled, etc - we can't block the main thread here and potentially resume building later.

Note that we do something similar for "Build and Run", where if there is an error it displays a dialog prompting how to fix, but in the end you have to do "Build and Run" again.

Are you thinking there is a way to package an "onSuccessAction" delegate that we can pass to these tool updaters that will continue the build bundle flow if the tool updates succeeded?

Given that, I think it at least makes sense to do these checks before asking for the save path.

Reply:
An "onSuccessAction" callback would be a nice way to implement it. 

My thinking is that because the save dialog doesn't show a title, users might not know what the dialog is referring to if it opens up after they've downloaded the bundletool. So if we run the tool check after the save dialog, when they return to the build flow the first thing they see is the appropriately titled progress bar.

I'm ok with the current functionality if we think this adds to much complexity for little benefit.

Reply:
I like the idea, but I need to do a bit of research. I'd like to move forward with the current approach, and come back to this idea in a subsequent PR.